src/main/java/net/snowflake/client/jdbc/SnowflakeStatementV1.java

```diff
@@ -887,12 +897,23 @@ private void executeSetProperty(final String sql) {
         }*/
       }
     } else {
-      this.sfStatement.executeSetProperty(sql);
+      this.sfBaseStatement.executeSetProperty(sql);
     }
   }
 
-  public SFStatement getSfStatement() throws SQLException {
-    return sfStatement;
+  public SFBaseStatement getStatementHandler() throws SQLException {
+    return sfBaseStatement;
+  }
+
+  // Convenience method to return an SFStatement-typed SFStatementInterface object, but
+  // performs the type-checking as necessary.
+  public SFStatement getSfStatement() throws SnowflakeSQLException {
```

Comment:
Could we make the naming style consistent with the `SFSession`? I don't have a preference but I believe consistency is crucial for improving the readability.

Reply:
I should rename `getStatementHandler` to `getSFBaseStatement` above, but I'm not sure that is what you mean. Could you clarify?

Reply:
If you mean in terms of upper/lower casing, `getSFSession()` is also shown with a lowercase `f` in `SnowflakeConnectionV1`. I don't want to change it to `getSFSession` and `getSFStatement` because it introduces a large amount of diffs (`getSfSession`) is the original casing.

Reply:
Sure, I feel there are two parts are a little bit weird for me:
1. upper/lower case for `getSfStaff` or `getSFStaff`?
2. Does it return base class or extended class for all `getSFStaff`?
Can you make them consistent across all newly introduced code? I don't have a preference which way is better though.

Reply:
The current way is extended class for all `getSfStuff()`. Base class can be fetched with `getSFBaseStuff()`.

Reply:
Sounds good! I assume you also change SFSession to apply this naming style.